tests/unit/adapter_tests.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright © 2016 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from cloudant.client import CouchDB
+from cloudant.adapters import Replay429Adapter
+from requests.packages.urllib3.util import Retry
+from .unit_t_db_base import UnitTestDbBase
+
+class AdapterTests(UnitTestDbBase):
+    """
+    Unit tests for transport adapters
+    """
+
+    def test_new_Replay429Adapter(self):
+        """
+        Test that a new Replay429Adapter is accepted as a parameter for a client.
+        """
+        try:
+            self.client = CouchDB(
+                self.user,
+                self.pwd,
+                url=self.url,
+                adapter=Replay429Adapter())
+        except Exception as err:
```

Comment:
Well the tests are not asserting that an exception is raised in this case. The objective was to fail if an exception is raised. In b6284d2 I've removed the try/except entirely - the test will still not pass in the case of an exception, but will be in `Error` state instead of failing.